src/MailerMessage.php

```diff
@@ -307,7 +319,88 @@ public function setHtmlBody($html)
         $this->_htmlBody = $html;
         return $this;
     }
-    
+
+    /**
+     * Get Custom Campaign
+     * @see https://dev.mailjet.com/guides/?shell#group-into-a-campaign
+     * @since 1.1.3
+     * 
+     * @return string CustomCampaign
```

Comment:
Please review *all* your phpdocs again:

+ it does not return, so no @return is needed.
+ No newslines between phpdoc tags

Please check the phpdoc guidline: https://github.com/luyadev/luya/blob/master/docs/guide/luya-guideline.md#phpdoc

Reply:
I have reviewed them. Hope it's ok like that ;-)